src/Module/Item/Display.php

```diff
@@ -136,7 +136,7 @@ protected function content(array $request = []): string
 		}
 
 		if ($item['gravity'] != Item::GRAVITY_PARENT) {
-			$parent = Post::selectFirstForUser($itemUid, $fields, [
```

Comment:
`selectFirstForUser` had been chosen deliberately since this prevents users from seeing posts from blocked accounts.

Reply:
I know, but doing this check at this level breaks the display that expects a top-level post.

Don't worry about the actual check, it is performed again on line 249 and then again in `Content\Conversation::addChildren`.